components/raftstore/src/store/worker/pd.rs

```diff
@@ -1468,6 +1487,67 @@ where
     fn handle_region_cpu_records(&mut self, records: Arc<RawRecords>) {
         calculate_region_cpu_records(self.store_id, records, &mut self.region_cpu_records);
     }
+
+    fn region_flow(&mut self, region_id: u64, written_bytes: u64, written_keys: u64) -> RegionFlow {
```

Comment:
If we report data at the region granularity, will it also cause too much heartbeat pressure?

Reply:
I think another problem with reporting at the region granularity may be that, as in previous versions, the reporting intervals and times for store and regions are inconsistent.